addons/website/tests/test_create_website.py

```diff
@@ -0,0 +1,9 @@
+from odoo.tests import HttpCase
+from odoo.tools import mute_logger
+
+class TestCreateWebsite(HttpCase):
+
+    def test_01_create_website(self):
+        with mute_logger('odoo.addons.base.models.ir_translation'):
+            self.env["base.language.install"].create({'lang': 'fr_FR', 'overwrite': True}).lang_install()
+        self.start_tour('/', 'test_create_website', login='admin')
```

Comment:
Missing return line, your IDE should tell you, easier to not forget that way ;-)